packages/frontend/src/components/atoms/form-field.atom.tsx

```
@@ -0,0 +1,32 @@
+ * FormField Atom Component
+ *
+ * Wrapper für Formularfelder mit Label und Hilfetexten
+ */
+export function FormField({ label, helperText, error, required, className, children, autoCorrect = 'off' }: FormFieldProps) {
+  return (
+    <div className={cn('space-y-2', className)}>
+      {label && (
+        <label autoCorrect={autoCorrect} className="block text-sm font-medium text-gray-700">
+          {label}
+          {required && <span className="ml-1 text-red-500">*</span>}
+        </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)